app/graphql/mutations/update_user.rb

```diff
@@ -15,9 +16,8 @@ class UpdateUser < GraphQL::Schema::Mutation
     def resolve(params)
       mutator = UpdateUserMutator.new(context, params)
 
-      success =
-        if mutator.valid?
-          mutator.notify(:success, I18n.t("shared.notifications.done"), I18n.t("mutations.update_user.profile_saved_notification"))
+      success = if mutator.valid?
+          mutator.notify(:success, I18n.t('shared.notifications.done'), I18n.t('mutations.update_user.profile_saved_notification'))
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)